### PR TITLE
turkish language file added.

### PR DESCRIPTION
**Pull request by melihucar**

### CODE REVIEW

File: src/gui/tr/turkish.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    remeshRemesh: 'Remesh',
+    remeshResolution: 'Çözünürlük',
+    remeshBlock: 'Engelle',
+
+    // dynamic
+    dynamicTitle: 'Dinamik Topoloji',
+    dynamicActivated: 'Aktifleştirildi (no quads)',
+    dynamicSubdivision: 'Altbölüm',
+    dynamicDecimation: 'Üstbölüm',
+    dynamicLinear: 'Doğrusal altbölüm',r
```
Comment (stephomi) [inline src/gui/tr/turkish.js]: Small typo

**Comment by stephomi** [comment]
Thanks!
I can merge when the typo is fixed, then I'll add the language to the gui.

**Comment by melihucar** [comment]
small typo fixed.

**Comment by stephomi** [comment]
Thanks!

**Pull request merged**